server/api.go

```diff
@@ -49,6 +54,30 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 	}
 }
 
+func (p *Plugin) disableForUser(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get("Mattermost-User-ID")
```

Comment:
On other plugins we usually check whether this is set, and return a "not authorized" error when this is not set. I see we are not doing this in this plugin at all.
The potential problem is someone not logged in hitting this endpoint, and doing all the operations over an empty string.

Reply:
good point. i'll add validation

Reply:
@larkox - actually, no need. Requests are wrapped in `requiresUserId` that does the validation

Reply:
I was looking for something like that specifically before making this comment, and I don't know how I didn't see it. :sweat_smile: